packages/card/src/js/index.ts

```diff
@@ -0,0 +1,7 @@
+export const toPercentageString = (num: number): string => {
+  try {
+    return Math.min(parseFloat(num.toFixed()), 100).toString() + '%'
+  } catch (_) {
```

Comment:
```suggestion
  } catch (_err) {
```

🤷  idk - seems like you'd need to do this to make ts happy